.gitmodules

```diff
@@ -68,3 +68,6 @@
 [submodule "examples/models/llama2/third-party/abseil-cpp"]
 	path = examples/models/llama2/third-party/abseil-cpp
 	url = https://github.com/abseil/abseil-cpp.git
+[submodule "third-party/ios-cmake"]
+	path = third-party/ios-cmake
+	url = https://github.com/leetal/ios-cmake
```

Comment:
is this necessary? 

in the phabricator version i didn't see these changes

Reply:
I guess when exported the stacked diffs get flattened in GitHub PRs.